nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -85,6 +85,8 @@ providers:
       - name: vyos-1.1.8-20190407
         config-drive: true
         connection-type: network_cli
+      - name: esxi-6.7.0-20190802001-STANDARD
+        config-drive: true
```

Comment:
we need to add the following to pools section below:

          - name: esxi-6.7.0
            flavor-name: s1.medium
            cloud-image: esxi-6.7.0-20190802001-STANDARD
            networks:
              - Public Internet
              - Private Network (10.0.0.0/8 only)
              - Private Network (Floating Public)